helsingborg-stad/modularity-guides
# 3.0/feature/accordion

Refactor script-based guide module using styleguide v2 components.

**Changelog**
- Replace card implementation with accordion from styleguide
- Refactor Prev/Next Button logic 
- Refactor mailing todo list feature
- Fix issues with Todo modal & z-index
- Add loader to email form
- Add notices to email form
- Render recaptcha using javascript

**Known bug/limitations**
- Does not lock next/prev button based on checked options
- Required checkbox feature not re-implemented
- Step 1 should be open initially but requires changes in component library and should be fixed outside of this PR

**How to test**
1. Build assets using npm run build
2. install & activate wordpress plugin
3. Enable module in modularity settings
4. Create a new guide
5. Test checkbox (should show different content & populate todolist)
6. Test pagination (should paginate when clicking next/prev button)
7. Test todo checklist email feature
